spec/system/candidate_interface/entering_details/degrees/candidate_entering_degree_with_predicted_grade_spec.rb

```diff
@@ -14,30 +14,32 @@ def given_i_am_signed_in
   end
 
   def when_i_add_a_degree_that_is_not_finished_yet
-    visit candidate_interface_application_form_path
-    click_link 'Degree'
-    choose 'UK degree'
-    fill_in 'Type of degree', with: 'Bachelor of Arts'
-    click_on_save_and_continue
-
-    fill_in 'What subject is your degree?', with: 'Fashion'
-    click_on_save_and_continue
-
-    fill_in 'Which institution did you study at?', with: 'Royal College of Art'
-    click_on_save_and_continue
-
-    expect(page).to have_content('Have you completed your degree?')
-    choose 'No'
-    click_on_save_and_continue
-
-    expect(page).to have_content('What grade do you think you’ll get?')
-    expect(page).to have_content('You must give an academic referee who can agree that you’re aiming for this grade.')
-    choose 'Third-class honours'
-    click_on_save_and_continue
-
-    fill_in 'Year started course', with: 2017
-    fill_in 'Graduation year', with: 2020
-    click_on_save_and_continue
+    Timecop.freeze(2020, 1, 1) do
```

Comment:
Needed to add a `Timecop.freeze` with the new validation but was required to roll the years forward (on lines 39 & 40) a year due to the changes made to CycleTimetable. Otherwise keeping it in 2019 it would result in the following:
```
     Failure/Error: schedule.fetch(name)

     ActionView::Template::Error:
       key not found: :show_deadline_banner
```

Reply:
Can we use RecruitmentCycle.current_year instead? So:

```
 fill_in 'Year started course', with: RecruitmentCycle.previous_year - 3
      fill_in 'Graduation year', with: RecruitmentCycle.previous_year 
      click_on_save_and_continue
      expect(page).to have_content('Enter a year that is in the future')

      fill_in 'Year started course', with: RecruitmentCycle.current_year - 3
      fill_in 'Graduation year', with: RecruitmentCycle.current_year
```